bower-polymer2.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "starcounter-include",
+  "authors": [
+    "Tomek Wytrebowicz <tomalecpub@gmail.com>"
+  ],
+  "homepage": "https://github.com/Starcounter/starcounter-include",
+  "description": "Custom Element to include HTML partials/templates from Starcounter",
+  "main": "starcounter-include.html",
+  "keywords": [
+    "template",
+    "partial",
+    "starcounter",
+    "web-components",
+    "include",
+    "shadow-dom",
+    "distribution",
+    "imported-template"
+  ],
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "Gruntfile.js",
+    "wwwroot/sys",
+    "test",
+    "wct.conf.json",
+    "demo",
+    "sc"
+  ],
+  "dependencies": {
+    "imported-template": "^2.0.0",
+    "translate-shadowdom": ">=0.0.4"
+  },
+  "devDependencies": {
+    "web-component-tester": "^6.0.0",
+    "polymer": "^2.0.0",
+    "document-register-element": "^1.5.0"
+  },
+  "variants": {
+    "polymer2": {
+      "directory": "bower_components-polymer2",
+      "devDependencies": {
+        "polymer": "^2.0.0"
+      }
+    }
+  },
+  "directory": "bower_components-polymer2"
+}
```

Comment:
Something went wrong with this file. You committed it to Git and added it to `.gitignore` simultaneously.